config/locales/provider_interface/make_offer.yml

```diff
@@ -8,28 +8,38 @@ en:
       providers:
         new:
           title: Training provider
+          caption: Make offer - %{name}
```

Comment:
could we make the captions to two single locations in the locales and choose the local we want in the views instead to avoid duplication?

Reply:
🆗